app/design/adminhtml/Magento/backend/Magento_ProductVideo/web/css/source/_module.less

```diff
@@ -197,7 +201,11 @@
                 height: 0;
 
                 .admin__field-control {
+                    /**
+                    *@codingStandardsIgnoreStart
+                    */
```

Comment:
The same here

Reply:
i'm afraid it's not work since magento validate less not allow use id. But in less you still allowed to use id in mixin
If you remove this line magento will show error static errors